src/main/java/Nick_Maven/Framework/service/ExpectedParamsCreatorService.java

```diff
@@ -0,0 +1,20 @@
+package Nick_Maven.Framework.service;
+
+import Nick_Maven.Framework.model.ParamsCloudGoogleCalculator;
+
+public class ExpectedParamsCreatorService {
+
+    private static final String EXPECTED_VM_CLASS = "testdata.ExpectedVM_CLASS";
+    private static final String EXPECTED_INSTANCE = "testdata.ExpectedINSTANCE";
+    private static final String EXPECTED_REGION = "testdata.ExpectedREGION";
+    private static final String EXPECTED_LOCAL_SSD = "testdata.ExpectedLOCAL_SSD";
+    private static final String EXPECTED_COMMITMENT_TERM = "testdata.ExpectedCOMMITMENT_TERM";
+    private static final String EXPECTED_TOTAL_PRICE = "testdata.ExpectedTOTAL_PRICE";
+
+    public static ParamsCloudGoogleCalculator withParamsFromProperty() {
```

Comment:
it would be better if the method name from a 'Factory' class combine particular staffs and retuning type (randomAccount) or fromPropertyParamsCloudGoogleCalculator for your case for example